src/components/textToSpeech/js/MockTTS.js

```diff
@@ -52,19 +52,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 funcName: "fluid.mock.textToSpeech.cancel",
                 args: ["{that}", "{that}.handleEnd"]
             },
-            pause: {
-                "this": null, // TODO: This needs to be removed once FLUID-5714 is fixed
-                method: null,
-                func: "{that}.events.onPause.fire"
-            },
-            resume: {
-                "this": null,
-                method: null,
-                func: "{that}.events.onResume.fire"
-            },
```

Comment:
They were removed because they refer to nonexistent events - I imagine @waharnum left them behind in the previous refactoring - for some reason, this did not trigger a failure in the previous framework